lib/bolt/cli.rb

```diff
@@ -212,12 +212,13 @@ def finalize_setup
 
     private def validate_ps_version
       if Bolt::Util.powershell?
-        target = inventory.get_target('localhost')
-        Bolt::Transport::Local.new.with_connection(target) do |conn|
-          # This will automatically validate the powershell version on the Bolt
-          # controller
-          Bolt::Shell::Powershell.new(target, conn)
-        end
+        stdout, _stderr, _status = Open3.capture3('pwsh -Command $PSVersionTable.PSVersion.Major')
```

Comment:
The way this currently works, the command is actually `powershell.exe -NoProfile -NonInteractive -NoLogo -ExecutionPolicy Bypass -Command $PSVersionTable.PSVersion.Major`.